concord_cli/tracer.py

```diff
@@ -32,6 +32,10 @@ def generate_options():
     return parser
 
 def validate_options(options, parser):
+    config = default_options()
+    if config is not None:
+        options.zookeeper = config['zookeeper_hosts']
+        options.zk_path = config['zookeeper_path']
```

Comment:
This code is repeated many times. Move into default_options and have it done once. Ye.